jinahub/indexers/searcher/FaissSearcher/tests/integration/test_train_index.py

```diff
@@ -79,3 +80,19 @@ def test_train_and_index(metas, tmpdir):
         assert len(result[0].matches) == 4
         for d in result:
             assert d.matches[0].scores['l2'].value >= d.matches[1].scores['l2'].value
+
+
+@pytest.mark.gpu
+@pytest.mark.docker
+def test_docker_runtime_gpu(build_docker_image_gpu: str):
+    with pytest.raises(subprocess.TimeoutExpired):
+        subprocess.run(
+            [
+                'jina',
+                'executor',
+                f'--uses=docker://{build_docker_image_gpu}',
+                '--gpus all',
```

Comment:
This should be `'--gpus', 'all'` - would you mind also fixing the documentation for it? We fixed this yesterday but forgot to update documentation

Reply:
Done